artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/ActiveMQSession.java

```diff
@@ -885,7 +891,7 @@ public TemporaryQueue createTemporaryQueue() throws JMSException {
       }
 
       try {
-         ActiveMQTemporaryQueue queue = ActiveMQDestination.createTemporaryQueue(this);
+         ActiveMQTemporaryQueue queue = ActiveMQDestination.createTemporaryQueue(this, (enable1xPrefixes ? PacketImpl.OLD_TEMP_QUEUE_PREFIX.toString() : ""));
```

Comment:
nit: but can avoid a string concatenation that occurs in the new  createTemporaryQueue method  for users not enabling, if the if was moved around, and the existing method called for that.

final ActiveMQTemporaryQueue queue;

if (enable1xPrefixes) {
 queue  = ActiveMQDestination.createTemporaryQueue(this, PacketImpl.OLD_TEMP_QUEUE_PREFIX.toString());
} else {
 queue  = ActiveMQDestination.createTemporaryQueue(this);
}